CODE_OF_CONDUCT.md

```diff
@@ -1,9 +1,14 @@
 # Contributor Covenant Code of Conduct
 
+> __PLEASE NOTE:__  
+> This document is the Official Covenant Code of Conduct that must be present in the root folder of _every_ devonfw project as `CODE_OF_CONDUCT.md`. Please, include this file (minus this note) in your repository and add the product owners email address in the place marked below.
+
```

Comment:
I do not get it. The idea of this .github repo is exactly that we do *not* have to copy the Code of Conduct to every repository as it is inherited automatically to all repos. Therefore this section does not make any sense to me.

Reply:
Ok, I'll remove this notice and just add our distribution list e-mail at the marked spot. Would that be o.k.?